deployment/quick-start/services-configuration.yaml.template

```diff
@@ -139,9 +139,3 @@ prometheus:
   ssh-secret-name: ssh-key-secret
   {% endif %}
   {% endif %}
-
-pylon:
```

Comment:
maybe just comment out all pylon default values, and leave a comments, so quick start users can simple uncomment value to change.